### ISSUE TITLE
[java] UseProperClassLoader can not detect the case with method call on intermediate variable

### ISSUE THREAD

**Issue by wuchiuwong** [pmd/pmd#2544]
**Affects PMD Version:** 
6.22.0

**Rule:**
[UseProperClassLoader](https://pmd.github.io/latest/pmd_rules_java_errorprone.html#useproperclassloader)

**Description:**
UseProperClassLoader can not detect the case which is written in two lines:<code>Foo foo = new Foo()</code> and then <code>ClassLoader classLoader = foo.getClassLoader();</code>
This rule is implemented through xpath search:

```
//PrimarySuffix[@Image='getClassLoader']
```
This rule can only detect the case <code>new Foo().getClassLoader()</code>

**Code Sample demonstrating the issue:**

```java
Foo foo = new Foo()
ClassLoader classLoader = foo.getClassLoader();
```

**Expected outcome:**
false-negative

**Running PMD through:**
CLI
